src/main/resources/pmd/pmd-ruleset.xml

```diff
@@ -41,12 +41,14 @@
     <exclude name="UseUnderscoresInNumericLiterals" />
     <exclude name="GenericsNaming" />
     <exclude name="ConfusingTernary" />
+    <exclude name="FieldDeclarationsShouldBeAtStartOfClass" />
```

Comment:
Is it the intention to put this back in the future?

Reply:
No. I disagree with this in some cases since I often find it helpful to have frequently used inner types (particularly public enums and interfaces) at the top of the class.